tests/input/relatedResource/relatedResource-missing-id-fail.json

```diff
@@ -0,0 +1,15 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2"
+  ],
+  "type": [
+    "VerifiableCredential"
+  ],
+  "credentialSubject": {
+    "id": "did:example:subject"
+  },
+  "relatedResource": [{
+    "url": "https://www.w3.org/ns/credentials/v2",
+    "digestMultibase": "uEres1usWcWCmW7uolIW2uA0CjQ8iRV14eGaZStJL73Vz"
```

Comment:
how did you produce digestMultibase here?

Reply:
@aljones15 I copied it from this example in the spec: https://w3c.github.io/vc-data-model/#example-usage-of-the-relatedresource-and-digestmultibase-property

It is likely wrong at this point...

I could pick a different resource served by this test suite to use instead. Sound better?

Reply:
If we go the route of including related resources in this repo for these tests, we will have to:
1. serve them with the report
2. prevent them from changing without updating the hash (probably via a README warning)
3. document how their hashes were created for anyone who wants to change them later